src/Migrator.EF6.Tools/DotnetToolDispatcher.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.PlatformAbstractions;
+using NuGet.Frameworks;
+
+namespace Microsoft.Extensions.Internal
+{
+	internal static class DotnetToolDispatcher
```

Comment:
Is there a reason for including this file? I see that it's internal, but dispatching is a standard task, isn't there a public api we can use?